.travis.yml

```diff
@@ -4,8 +4,8 @@ python:
   - 3.5
   - "nightly"
 install:
-  - pip install -r requirements-tests.txt
+  - pip install --upgrade -r requirements-tests.txt
 script:
-  - coverage run --source xonsh -m py.test -q
+  - py.test --flake8 --cov=. --cov-report=term
```

Comment:
i think this should be `--cov=xonsh` then you can stop omitting tests, setup etc on .coveragerc